### PR TITLE
Adds a tunnel clown syndie bundle

### PR DESCRIPTION
**Pull request by jughu**
THE PREDATORS OF MAINTENCE RETURNS

:cl: Jughu
add: Adds tunnelclown syndie bundle
/:cl:

### CODE REVIEW

**Comment by scoopscoop** [comment]
I don't know, this seems like something you'd just hate to get since it's obvious as hell and not that powerful

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -144,6 +144,15 @@
...
 			new /obj/item/device/chameleon(src) // 7 tc
+			
+		if("tclown") // Tunnelclowns the predetors of maint
+			new /obj/item/clothing/glasses/thermal/monocle(src) // 4tc since copypasta thermals
+			new /obj/item/clothing/suit/hooded/chaplain_hoodie(src)
+			new /obj/item/clothing/shoes/clown_shoes(src)
+			new /obj/item/weapon/twohanded/required/chainsaw(src)
+			new /obj/item/clothing/mask/gas/clown_hat(src)
+			new /obj/item/weapon/bikehorn(src)
+			new /obj/item/clothing/gloves/color/black(src)
```
Comment (Supermichael777) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: make these black combat gloves

**Comment by Durkel** [comment]
This is a booby prize if you get it, all the gear you get doesn't even equal 20tc.

**Comment by jughu** [comment]
I will work tommorow at i have a idea to make it worth

**Comment by jughu** [comment]
What if i add a demon heart to it :^)

**Comment by Durkel** [comment]
You can't blood jaunt with a weapon :^). rip chainsaw.

**Comment by jughu** [comment]
I could make it a fireaxe again you can put that on your back its the orginal tunnel clown anyways just creepier with the blood jaunt and scarier

**Comment by bgobandit** [comment]
add a chameleon projector, you're walking around an empty maint tunnel and all of a sudden WILD CLOWN APPEARS

**Comment by jughu** [comment]
Look this is the feedback i need why cant everyone be like this :)

**Comment by lordpidey** [comment]
What if you gave it a custom demon heart that allows blood jaunting while holding the chainsaw?

**Comment by jughu** [comment]
Then people  will just ditch the chainsaw and grab tasers or something and jaunt around and fire axe is the orginal tunnel clown anyway

**Comment by jughu** [comment]
now i need real feedback OK so either cham projector or demon heart idk tho WHICH
so vote with ur frickin reactions ok 
heart for demon heart and laughing for cham projector

**Comment by Durkel** [comment]
Chamo projector makes the most sense, but it still leaves the bundle rather weak.

A demon heart is incredibly strong, even after the nerf, Perhaps too much so for it be a option. That said, the demon heart is the most "scary" and does make some sense. Honestly, you should open up a poll on the forum or ask in irc for opinions.

**Comment by jughu** [comment]
You are Right

**Comment by lordpidey** [comment]
By custom demon heart, I meant one that allows jaunting with the chainsaw held, and nothing else.

**Comment by jughu** [comment]
FIRE AXE OK

**Comment by optimumtact** [comment]
Is this ready?

**Comment by jughu** [comment]
No NO

**Comment by optimumtact** [comment]
hurry up

**Comment by jughu** [comment]
I have to add a additional thing ok

**Comment by jughu** [comment]
I close this for now u reopen when the thing gets addrd

**Pull request closed without merging**